src/components/Supporters/index.jsx

```diff
@@ -0,0 +1,227 @@
+    },
+];
+
+const infrastructure = [
+    {
+        name: "Anaconda",
+        link: "https://www.anaconda.com/",
+        light: "img/supporters/anaconda_light.svg",
+        dark: "img/supporters/anaconda_dark.svg",
+        width: 250,
+    },
+    {
+        name: "Open Source Lab",
+        link: "https://osuosl.org/",
+        light: "img/supporters/osl_light.svg",
+        dark: "img/supporters/osl_dark.svg",
+        width: 150,
+    },
+    {
+        name: "OVHcloud",
+        link: "https://www.ovhcloud.com/",
+        light: "img/supporters/ovh_light.svg",
+        dark: "img/supporters/ovh_dark.svg",
+        width: 250,
+    },
+    {
+        name: "Digital Ocean",
+        link: "https://www.digitalocean.com/",
+        light: "img/supporters/do_light.svg",
+        dark: "img/supporters/do_dark.svg",
+        width: 250,
+    },
+    {
+        name: "Oracle Cloud",
+        link: "https://www.oracle.com/cloud/",
+        light: "img/supporters/oracle_light.svg",
+        dark: "img/supporters/oracle_dark.svg",
+        width: 200,
+    },
+];
+
+const developer = [
+    {
+        name: "Anaconda",
+        link: "https://www.anaconda.com/",
+        light: "img/supporters/anaconda_light.svg",
+        dark: "img/supporters/anaconda_dark.svg",
+        width: 250,
+    },
+    {
+        name: "Nvidia",
+        link: "https://www.nvidia.com/",
+        light: "img/supporters/nvidia_light.svg",
+        dark: "img/supporters/nvidia_dark.svg",
+        width: 250,
+    },
+    {
+        name: "Voltron Data",
+        link: "https://voltrondata.com/",
+        light: "img/supporters/voltron_light.svg",
+        dark: "img/supporters/voltron_dark.svg",
+        width: 250,
+    },
+    {
+        name: "Quansight Labs",
+        link: "https://labs.quansight.org/",
+        light: "img/supporters/quansight_light.svg",
+        dark: "img/supporters/quansight_dark.svg",
+        width: 250,
+    },
+    {
+        name: "QuantStack",
+        link: "https://quantstack.net/",
+        light: "img/supporters/quantstack_light.svg",
+        dark: "img/supporters/quantstack_dark.svg",
+        width: 250,
+    },
+];
+
+export default function Supporters() {
+    return (
+        <div className={[styles.supporters, styles.section_padding].join(" ")}>
+            <div className={styles.supporters_conda_forge}>
+                <h1>Supporters</h1>
+                <p>
+                    If you like conda-forge and want to support our mission,
+                    please consider making a{" "}
+                    <Link to="https://opencollective.com/conda-forge">
+                        donation
+                    </Link>{" "}
+                    to support our efforts.
+                </p>
+            </div>
+            <div className={styles.fiscal_sponsor}>
+                <div className={styles.numfocus_card}>
+                    <Link to="https://numfocus.org/">
+                        <ThemedImage

```

Comment:
Does `ThemedImage` support alt text? I can only hope so because we REALLY need to have that. I'd add it to the data fields above and then expose it here. You can read about good alt texts in this resource by Harvard: [Write good Alt Text to describe images](https://accessibility.huit.harvard.edu/describe-content-images)

Reply:
`ThemedImage` does support alt text. The name of the company is the alt text for all the images. It is implemented like this:

```
<ThemedImage
   className={styles.image}
   alt={name}
   sources={{
   light: useBaseUrl(`${light}`),
   dark: useBaseUrl(`${dark}`),
   }}
 width={width}
/>
```

Reply:
I changed `alt` to something like this `alt={${name} logo}`. This will add the word `logo` in front of each brand's name